packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -32,6 +32,7 @@
   "Accounts_AllowRealNameChange": "Allow Name Change",
   "Accounts_AllowUsernameChange": "Allow Username Change",
   "Accounts_AllowUserProfileChange": "Allow User Profile Change",
+  "Accounts_AvatarCacheTime": "Avatar cache time(by http protocol)",
```

Comment:
Please remove `(by http protocol)` from the name and user the description aspect of the settings.